benches/flamegraph.rs

```diff
@@ -0,0 +1,37 @@
+extern crate criterion;
+extern crate inferno;
+
+use criterion::*;
+use inferno::flamegraph;
+use std::fs::File;
+use std::io::{BufReader, Cursor, Read};
+
+fn flamegraph_benchmark(c: &mut Criterion) {
+    let mut f = File::open("tests/data/collapse-perf/results/example-perf-stacks-collapsed.txt")
```

Comment:
I wonder if it might make sense to also define a second, identical benchmark that uses the collapsed DTrace output? Just so we have two data-points instead of one. Alternatively, we could even flamegraph across the concatenation of the two.